databases/week3/hyf_meal_sharing_db.sql

```diff
@@ -0,0 +1,155 @@
+-- CREATE database hyf_meal_sharing_db;
+
+use hyf_meal_sharing_db;
+
+CREATE table `meals`(
```

Comment:
Overall you forgot to specify the character set when creating the tables.

Reply:
Thank you a lot for all your comments, they were really helpful. I made all the changes you suggested.